src/reducers/app/reducers.ts

```diff
@@ -121,10 +121,9 @@ const appReducer = (state = initialState, action) => {
     case SET_ITEMS:
       return { ...state, [action.entity]: action.items };
     case SET_COLUMNS: {
-      const columns = state.columns;
-      const newColumns = { ...columns };
-      newColumns[action.entity] = action.items;
-      return { ...state, columns: newColumns };
+      const columns = {...state.columns};
```

Comment:
Why not `const columns = state.columns;`?

Reply:
`const columns = state.columns;`
The reference of `columns` and `state.columns` is the same.
That is, no new variables were actually created.
Destructuring assignment creates a new variable.